tests/Feature/Nova/OrderResourceTest.php

```diff
@@ -24,5 +31,42 @@ public function index()
             ->assertOk();
 
         $this->assertCount(4, $response->json('resources'));
+
+        $orderFields = collect($response->json('resources')[0]['fields'])->pluck('attribute');
+
+        $this->assertCount(4, $orderFields);
+        $this->assertEquals([
+            'id',
+            'order_number',
+            'amount',
+            'created_at',
```

Comment:
Without stub resources, location and customer don't appear in the response because those resources aren't defined.